.travis.yml

```diff
@@ -41,10 +39,9 @@ install:
     - echo $PATH
     - bids-validator --version
     - which bids-validator
-    - conda create -n testenv --yes python=${PYTHON_VERSION}
+    - conda env create -f environment.yml --name=testenv
     - source activate testenv
-    - conda install --yes --quiet numpy scipy matplotlib
-    - pip install flake8 pytest pytest-cov pytest-sugar pytest-faulthandler check-manifest
+    - pip uninstall mne  # uninstall the pip version of mne to install dev version below
```

Comment:
you don't need to do this actually but it's fine

Reply:
okay, good to know! I'll leave it: explicit > implicit